python/src/trezorlib/cardano.py

```diff
@@ -370,4 +370,17 @@ def sign_tx(
         )
     )
 
-    return response
+    result = bytearray()
+    while isinstance(response, messages.CardanoSignedTxChunk):
+        result.extend(response.signed_tx_chunk)
+        response = client.call(messages.CardanoSignedTxChunkAck())
+
+    if not isinstance(response, messages.CardanoSignedTx):
+        raise exceptions.TrezorException("Unexpected response: {response}")
```

Comment:
this will show the literal string "{response}" :)
probably best to just say "unexpected response", a normal user won't be able to decode any sort of detailed information anyway, and a power-user can dump the protobuf messages in a cleaner way